contracts/StaderOracle.sol

```diff
@@ -29,7 +29,7 @@ contract StaderOracle is IStaderOracle, AccessControlUpgradeable, PausableUpgrad
     uint256 public constant MAX_ER_UPDATE_FREQUENCY = 7200 * 7; // 7 days
     uint256 public constant ER_CHANGE_MAX_BPS = 10_000;
     uint256 public override erChangeLimit;
-    uint256 public constant MIN_TRUSTED_NODES = 5;
```

Comment:
is this affecting any other usecase ? Do we require 5 nodes atleast on any usecase here ?

Reply:
No, we decide to reduce the number of ONO's to the 3, and this strict check is preventing us to do so.

Reply:
cool